src/adapters/adaptersFactory.ts

```diff
@@ -0,0 +1,20 @@
+import { SocialNetworkOauth2AdapterInterface } from './oauth2/SocialNetworkOauth2AdapterInterface';
+import { DiscordAdapter } from './oauth2/discordAdapter';
+import { SOCIAL_NETWORKS } from '../entities/socialProfile';
+import { errorMessages } from '../utils/errorMessages';
+import { GoogleAdapter } from './oauth2/googleAdapter';
+
+const discordAdapter = new DiscordAdapter();
+const googleAdapter = new GoogleAdapter();
+export const getSocialNetworkAdapter = (
+  socialNetwork: string,
+): SocialNetworkOauth2AdapterInterface => {
```

Comment:
We are missing the LINKEDIN adapter here.

Reply:
Oh nice catch, I just forgot to add that